mssw/Acme-Spinodal-Utils/t/Utils.t

```diff
@@ -0,0 +1,51 @@
+package t::Acme::Spinodal::Utils;
+
+use strict;
+use warnings;
+use v5.12;
+
+use Data::Dumper;
+
+use Test::More;
+use Test::Exception;
+
+BEGIN {
+  use_ok( 'Acme::Spinodal::Utils' ) || print "Bail out!\n";
+}
+
+diag("Testing Util $Acme::Spinodal::Utils::VERSION, Perl $], $^X");
+
+ok( defined &Acme::Spinodal::Utils::sum, '&Acme::Spinodal::Utils::sum is defined' );
+
+{    # sum tests
+    { # No args
+    my $total;
+        eval { $total = Acme::Spinodal::Utils::sum() };
+        my $err = $@;
+        is( $total, 0, "Checking that the total is zero when no args are given." );
+        is( $err, '', "Checking no error was returned.")
+    }
+    
+    { # Expected total
+    my $total;
+        eval { $total = Acme::Spinodal::Utils::sum( qw(1 2 3 4) ) };
+        my $err = $@;
+        is( $total, 10, "Checking that the correct total is returned." );
+        is( $err, '', "Checking no error was returned.")
```

Comment:
Bit unnecessary as `$total` is defined. The whole test could just have been:

```perl
is ( sum( qw(1 2 3 4) ), 10, 'Integers sum correctly' );
```

Or this, if you're truly worried about `die`'s:
```perl
is ( eval { sum( qw(1 2 3 4) ) }, 10, 'Integers sum correctly' );
```